modules/auxiliary/scanner/http/epmp1000_dump_hashes.rb

```diff
@@ -104,10 +104,7 @@ def hash_dump(config_uri, cookie)
       )
 
       if good_response
-        vprint_status('++++++++++++++++++++++++++++++++++++++')
         vprint_status("#{rhost}:#{rport} - Dumping password hashes")
```

Comment:
I think this should be changed to print_status.

Reply:
Same as [this comment](https://github.com/rapid7/metasploit-framework/pull/9316#discussion_r158294341) -- you can end up with randomly placed blocks of plusses in a very responsive scan run.